### ISSUE TITLE
Spyder startup failure

### ISSUE THREAD

**Issue by PLinzer** [spyder-ide/spyder#16825]
### What steps will reproduce the problem?

On every Spyder 5.0.0 startup on Win10 Pro 64-bit.

### Traceback
```python-traceback
--- Logging error ---
Traceback (most recent call last):
  File "C:\Anaconda3\lib\asyncio\windows_events.py", line 453, in finish_recv
    return ov.getresult()
OSError: [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\asyncio\proactor_events.py", line 768, in _loop_self_reading
    f.result()  # may raise
  File "C:\Anaconda3\lib\asyncio\windows_events.py", line 808, in _poll
    value = callback(transferred, key, ov)
  File "C:\Anaconda3\lib\asyncio\windows_events.py", line 457, in finish_recv
    raise ConnectionResetError(*exc.args)
ConnectionResetError: [WinError 995] The I/O operation has been aborted because of either a thread exit or an application request

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Anaconda3\lib\logging\__init__.py", line 1084, in emit
    stream.write(msg + self.terminator)
AttributeError: 'NoneType' object has no attribute 'write'
Call stack:
  File "C:\Anaconda3\Scripts\spyder-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\Anaconda3\lib\site-packages\spyder\app\start.py", line 214, in main
    mainwindow.main(options, args)
  File "C:\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2293, in main
    mainwindow = create_window(app, splash, options, args)
  File "C:\Anaconda3\lib\site-packages\spyder\app\mainwindow.py", line 2192, in create_window
    app.exec_()
  File "C:\Anaconda3\lib\site-packages\spyder\plugins\application\container.py", line 219, in _check_updates_ready
    box.exec_()
  File "C:\Anaconda3\lib\site-packages\qtconsole\manager.py", line 27, in poll
    super().poll()
  File "C:\Anaconda3\lib\site-packages\jupyter_client\restarter.py", line 106, in poll
    if not self.kernel_manager.is_alive():
  File "C:\Anaconda3\lib\site-packages\jupyter_client\utils.py", line 23, in wrapped
    return loop.run_until_complete(future)
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 64, in run_until_complete
    self._run_once()
  File "C:\Anaconda3\lib\site-packages\nest_asyncio.py", line 100, in _run_once
    handle._run()
  File "C:\Anaconda3\lib\asyncio\events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "C:\Anaconda3\lib\asyncio\proactor_events.py", line 776, in _loop_self_reading
    self.call_exception_handler({
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1733, in call_exception_handler
    self.default_exception_handler(context)
  File "C:\Anaconda3\lib\asyncio\base_events.py", line 1707, in default_exception_handler
    logger.error('\n'.join(log_lines), exc_info=exc_info)
Message: 'Error on reading from the event loop self pipe\nloop: <ProactorEventLoop running=True closed=False debug=False>'
Arguments: ()
```

## Versions

* Spyder version: 5.0.0 
* Python version: 3.8.5
* Qt version: 5.12.9
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0          :  1.4.0 (OK)
chardet >=2.0.0               :  4.0.0 (OK)
cloudpickle >=0.5.0           :  2.0.0 (OK)
cookiecutter >=1.6.0          :  1.7.0 (OK)
diff_match_patch >=20181111   :  20200713 (OK)
intervaltree >=3.0.2          :  3.0.2 (OK)
IPython >=7.6.0               :  7.29.0 (OK)
jedi =0.17.2                  :  0.17.2 (OK)
jsonschema >=3.2.0            :  4.2.1 (OK)
keyring >=17.0.0              :  23.2.1 (OK)
nbconvert >=4.0               :  6.3.0 (OK)
numpydoc >=0.6.0              :  1.1.0 (OK)
paramiko >=2.4.0              :  2.8.0 (OK)
parso =0.7.0                  :  0.7.0 (OK)
pexpect >=4.4.0               :  4.8.0 (OK)
pickleshare >=0.4             :  0.7.5 (OK)
psutil >=5.3                  :  5.8.0 (OK)
pygments >=2.0                :  2.10.0 (OK)
pylint >=1.0                  :  2.11.1 (OK)
pyls >=0.36.2;<1.0.0          :  0.36.2 (OK)
pyls_black >=0.4.6            :  0.4.6 (OK)
pyls_spyder >=0.3.2           :  0.3.2 (OK)
qdarkstyle =3.0.2             :  3.0.2 (OK)
qstylizer >=0.1.10            :  0.2.1 (OK)
qtawesome >=0.5.7             :  1.1.0 (OK)
qtconsole >=5.0.3             :  5.1.0 (OK)
qtpy >=1.5.0                  :  1.11.2 (OK)
rtree >=0.8.3                 :  0.8.3 (OK)
setuptools >=39.0.0           :  59.1.0 (OK)
sphinx >=0.6.6                :  4.3.0 (OK)
spyder_kernels >=2.0.1,<2.1.0 :  2.0.5 (OK)
textdistance >=4.2.0          :  4.2.1 (OK)
three_merge >=0.1.1           :  0.1.1 (OK)
watchdog >=0.10.3;<2.0.0      :  1.0.2 (OK)
zmq >=17                      :  22.3.0 (OK)

# Optional:
cython >=0.21                 :  0.29.24 (OK)
matplotlib >=2.0.0            :  3.4.3 (OK)
numpy >=1.7                   :  1.20.1 (OK)
pandas >=1.1.1                :  1.3.4 (OK)
scipy >=0.17.0                :  1.7.2 (OK)
sympy >=0.7.3                 :  1.9 (OK)
```

**Comment by ccordoba12** [comment]
Hey @PLinzer, thanks for reporting. This problem should fixed in our 5.0.5 version. Please update by closing Spyder, opening the Anaconda Prompt and running there

    conda update anaconda
    conda install spyder=5.0.5

Let us know if it works for you.

**Comment by ccordoba12** [comment]
After those commands, please also run

    conda install jupyter_client=6.1.12

That could help too.

**Comment by ccordoba12** [comment]
Closing due to lack of response.

**Comment by PLinzer** [comment]
Sorry, I was expecting to receive e-mail notifications of responses.  The only notification I received was that this issue was closed earlier today.

In any event, I have uninstalled Anaconda and installed WinPython (Python v3.9.5 64-bit) and upgraded to Spyder v5.2.0.  While there's an apparent version incompatibility with PyQt5 (I had to downgrade to v5.12.3), I've got everything working correctly. now.

Thanks anyway.
